external/boost/CMakeLists.txt.boost.in

```diff
@@ -30,6 +30,7 @@ ExternalProject_Add(boost-download
                         libs/container_hash
                         libs/core
                         libs/detail
+                        libs/filesystem
```

Comment:
This is not good, that we take in extra boost filesystem library just for tests. Even now we have a lot of dependencies and checking out boost repo takes ages. Maybe we can add it only when tests are enabled. Is this possible?